README.md

```diff
@@ -42,6 +42,17 @@ sources is preferred, use `committed-src` as the hook id instead.
 Be sure to change `rev` to use the desired `committed` git tag or
 revision.
 
+By default, the `pre-commit install` only installs hooks of the `pre-commit` type,
+so it may be necessary to list the `commit-msg` hook during installation
+
+```bash
+pre-commit install -t pre-commit -t commit-msg
+```
+
+or require it using the `default_install_hook_types` configuration setting.
+
+https://pre-commit.com/#top_level-default_install_hook_types
```

Comment:
Should we instead have the example config use `default_install_hook_types` with a note about why its there?

Reply:
Yes. I've just updated my changes.